thevoid/connection_p.hpp

```diff
@@ -213,6 +228,12 @@ class connection : public std::enable_shared_from_this<connection<T>>, public re
 	const char *m_unprocessed_end;
 
 	bool m_pause_receive;
+
+	std::chrono::steady_clock::time_point m_receive_start;
+	std::chrono::microseconds m_receive_time;
+
+	std::chrono::steady_clock::time_point m_send_start;
```

Comment:
Are you really need m_send_start as a class' member? Why it shouldn't be created as a local variable in place it is used and be binded into callback?
The same question for m_receive_start.